README.md

```diff
@@ -42,7 +42,15 @@ represent the tiles.
 
 ## A1.1
 
-«replace this with your answer»
+S -> C
+
+S -> C S C
+
+C -> "r"
+
+C -> "g"
+
+C -> "b"
```

Comment:
You're on the right track, but the rule "S -> C S C" doesn't do what you want, because the two Cs could match different colors.

# 2

I was looking for something like:

~~~
       S -> r | g | b
       S -> r  S  r
       S -> g  S  g
       S -> b  S  b
~~~